### PR TITLE
Docs: add tabbing between psithon and psi api

### PR DESCRIPTION
**Pull request by philipmnel**
## Description

- [x] Fix issue in syntax highlighting for psithon code snippets - there are bits that are invalid python and we don't want highlighted as errors.
- [x] Add tabbing between Psithon and PsiAPI code snippets in the tutorial on input files.
 
## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

**Comment by philipmnel** [comment]
CI failure seems to be unrelated error in compiling Psi4? The docs do build successfully on my machine.

**Comment by TiborGY** [comment]
> CI failure seems to be unrelated error in compiling Psi4? The docs do build successfully on my machine.

The docs CI seems to be flaky at the moment, no idea why

**Comment by loriab** [comment]
It looks like I forgot to add `CMAKE_BUILD_PARALLEL_LEVEL=2` to the docs CI. I learned that when a Ninja build of Psi4 gives up in the early 200s of 1200 files, it's because it's out of memory on CI nodes when compiling libmints. Ninja automatically oversubscribes the available cores, so throttling down to allocated cores helps. Docs were recently converted to Ninja CMake generator (from Makefiles CMake generator).

File: doc/sphinxman/source/psithoninput.rst [hunk 1]
```diff
@@ -246,16 +254,29 @@ processed after the CCSD computation completes.
...
-   })
+.. tabs::
+
+   .. code-tab:: py PSIthon
+
+        set {
+            scf_type = pk
+            e_convergence =  1.e-5
+            soscf = True
+            optking__geom_maxiter = 50
```
Comment (loriab) [inline doc/sphinxman/source/psithoninput.rst:269]:
```suggestion
        }
        set {
            geom_maxiter = 50
```

Comment (loriab) [reply doc/sphinxman/source/psithoninput.rst:269]: unfortunately `<module>__` doesn't work in psithon.

File: doc/sphinxman/source/psithoninput.rst [hunk 2]
```diff
@@ -273,27 +294,53 @@ PSI Variables
...
+        molecule h {
+          H
+        }
+        
+        set basis cc-pvdz
+        set reference uhf
+        h_energy = energy('scf')
+        
+        D_e = psi_hartree2kcalmol * (2*h_energy - h2_energy)
+        print "De = %f" % D_e
```
Comment (loriab) [inline doc/sphinxman/source/psithoninput.rst:321]:
```suggestion
        print("De = %f" % D_e)
```
wow, it has been awhile :-)

File: doc/sphinxman/source/psithoninput.rst [hunk 3]
```diff
@@ -303,12 +350,21 @@ energy and printed to the output file using standard Python notation.
 Generally, there are multiple quantities of interest. Appendix
 :ref:`apdx:psivariables_module` lists PSI variables variables set by each
 module, and :ref:`apdx:psivariables_alpha` defines them.  These can be
-accessed through the :py:func:`~psi4.core.get_variable` function. For example, after
+accessed through the :py:func:`~psi4.core.variable` function. For example, after
```
Comment (loriab) [inline doc/sphinxman/source/psithoninput.rst:353]: good catch

File: doc/sphinxman/source/psithoninput.rst [hunk 4]
```diff
@@ -481,23 +654,43 @@ the |PSIfour| suite.
...
-  energy('mp2/cc-pv[dt]z')
+      energy('mp2', corl_basis='cc-pv[dt]z', corl_scheme=corl_xtpl_helgaker_2)
+
+      energy('mp2/cc-pv[dt]z')
+
+   .. code-tab:: py PsiAPI
+
+      # all equivalent
+
+      psi4.energy('mp2', corl_basis='cc-pv[dt]z', corl_scheme=corl_xtpl_helgaker_2)
```
Comment (loriab) [inline doc/sphinxman/source/psithoninput.rst:691]:
```suggestion
      energy('mp2', corl_basis='cc-pv[dt]z', corl_scheme='corl_xtpl_helgaker_2')

      energy('mp2/cc-pv[dt]z')

   .. code-tab:: py PsiAPI

      # all equivalent

      psi4.energy('mp2', corl_basis='cc-pv[dt]z', corl_scheme='corl_xtpl_helgaker_2')
```
this changed with DDD, I think

**Review by loriab — COMMENTED** [review]
after a few tweaks, lgtm, thanks! I think this'll be very helpful in spreading PsiAPI.

**Review by loriab — APPROVED** [review]
lgtm, thanks!

**Review by jturney — APPROVED** [review]

**Pull request merged**